apps/design/backend/src/worker/generate_election_package.ts

```diff
@@ -105,6 +90,7 @@ export async function generateElectionPackage(
     ballotMode: 'test',
     layoutOptions,
     nhCustomContent,
+    translatedElectionStrings: electionStrings,
```

Comment:
This will get used in the next commit, to add translations to the CDF ballot definition in the election package. And down the road, will also be available for multi-language HMPBs